api/src/test/java/org/pmiops/workbench/conceptset/ConceptSetMapperTest.java

```diff
@@ -0,0 +1,71 @@
+package org.pmiops.workbench.conceptset;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.sql.Timestamp;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.api.Etags;
+import org.pmiops.workbench.db.model.CommonStorageEnums;
+import org.pmiops.workbench.db.model.DbConceptSet;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.model.ConceptSet;
+import org.pmiops.workbench.model.Domain;
+import org.pmiops.workbench.model.Surveys;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+public class ConceptSetMapperTest {
+
+  @Autowired private ConceptSetMapper conceptSetMapper;
+
+  private DbConceptSet dbConceptSet;
+
+  @TestConfiguration
+  @Import({ConceptSetMapperImpl.class})
+  static class Configuration {}
+
+  @Before
+  public void setUp() {
+    Timestamp now = new Timestamp(System.currentTimeMillis());
+    DbUser creator = new DbUser();
+    creator.setUsername("brian");
+    dbConceptSet =
+        DbConceptSet.builder()
+            .addConceptSetId(1)
+            .addCreationTime(now)
+            .addCreator(creator)
+            .addDescription("descr")
+            .addDomain(CommonStorageEnums.domainToStorage(Domain.CONDITION))
+            .addLastModifiedTime(now)
+            .addName("conceptSet")
+            .addParticipantCount(200)
+            .addSurvey(CommonStorageEnums.surveysToStorage(Surveys.THE_BASICS))
+            .addVersion(1)
+            .addWorkspaceId(1)
+            .build();
+  }
+
+  @Test
+  public void dbModelToClient() {
```

Comment:
Can you test the other direction? Or is that not implemented yet?

Reply:
It's not implemented yet. I created a ticket for it: https://precisionmedicineinitiative.atlassian.net/browse/RW-4455
It's a little more involved than just copy over ConceptSet to DbConceptSet. So I want to think about it a bit... needs to hit BQ with proper OMOP table to get counts.